perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -284,10 +310,22 @@ Same as Symology above, but different attribute name.
 
     # The title field gets pushed to location fields in Echo/Symology, so include closest address
     my $title = $row->title;
-    if ($h->{closest_address}) {
-        if (my $addr = $h->{closest_address}{name} || $h->{closest_address}{display_name} || '') {
-            $addr =~ s/, England, United Kingdom$//;
-            $title .= "; Nearest calculated address = $addr";
+    if ( $h->{closest_address} ) {
```

Comment:
As mentioned in Slack at https://mysociety.slack.com/archives/C01TK8P1K8T/p1689929218707969?thread_ts=1689928394.321709&cid=C01TK8P1K8T, there is a `->summary` call on the Address objects that returns the information needed here, so that would be what to use, and this code won't need to know which geocoder is being used.

Reply:
Could also add a bit of POD here about this in addition